daemon/src/maker_cfd_actor.rs

```diff
@@ -292,24 +301,136 @@ impl MakerCfdActor {
         let cfd = Cfd::new(
             current_order.clone(),
             msg.quantity,
-            CfdState::Accepted {
+            CfdState::IncomingOrderRequest {
                 common: CfdStateCommon {
                     transition_timestamp: SystemTime::now(),
                 },
+                taker_id: msg.taker_id,
             },
         );
         insert_cfd(cfd, &mut conn).await?;
 
+        self.cfd_feed_actor_inbox
+            .send(load_all_cfds(&mut conn).await?)?;
+
```

Comment:
I think we should set `current_order_id` to `None` here otherwise a 2nd taker can override someone else's order.